Drop unused imports

r-barnes opened the pull request:
Summary:
From
```
./python/libcst/libcst codemod remove_unused_imports.RemoveUnusedImportsWithGlean --no-format caffe2/
```

Test Plan: Standard sandcastle tests

Differential Revision: D25727352

pritamdamania87 commented on torch/nn/modules/utils.py:
```diff
@@ -1,4 +1,3 @@
-from typing import List
```
Looks like this change is causing a lot of test failures.

pritamdamania87 reviewed (changes requested):
Thanks for putting up this PR! Although, there are a lot of CI failures, could you fix those?

r-barnes committed:
[Don't review] Drop unused imports (#49972)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/49972

From
```
./python/libcst/libcst codemod remove_unused_imports.RemoveUnusedImportsWithGlean --no-format caffe2/
```

Test Plan: Standard sandcastle tests

Differential Revision: D25727352

fbshipit-source-id: 40a2e44a05e4525404cd1aa5422ce3624acf2702
